core/tx_pool.go

```diff
@@ -55,6 +56,25 @@ var (
 	evictionInterval     = time.Minute   // Time interval to check for evictable transactions
 )
 
+var (
+	// Metrics for the pending pool
+	pendingDiscardMeter    = metrics.NewMeter("txpool/pending/discard") 
+	pendingReplaceMeter    = metrics.NewMeter("txpool/pending/replace")
+	pendingRLMeter         = metrics.NewMeter("txpool/pending/ratelimit") // Dropped due to rate limiting
+	pendingNofundsMeter    = metrics.NewMeter("txpool/pending/nofunds")   // Dropped due to out-of-funds
```

Comment:
You forgot to add the metrics collection for this. Should be added around line 670.